rpcserver.go

```diff
@@ -3683,6 +3683,17 @@ func handleVerifyMessage(s *rpcServer, cmd interface{}, closeChan <-chan struct{
 	return address.EncodeAddress() == c.Address, nil
 }
 
+// handleEstimateFee handles estimatefee commands.
```

Comment:
And move this up to the after the `handleDecodeScript` function so it is alphabetized as the rest of them are.